morituri/common/program.py

```diff
@@ -257,33 +257,24 @@ def getPath(self, outdir, template, mbdiscid, i, profile=None,
         template = re.sub(r'%(\w)', r'%(\1)s', template)
 
         ret = os.path.join(outdir, template % v)
-
-
-
         return ret
 
-    def getCDDB(self, cddbdiscid):
-        """
-        @param cddbdiscid: list of id, tracks, offsets, seconds
+    def craftMusicBrainzFromFreeDB(self, freedb_data):
```

Comment:
A short docstring explaining that this is creating musicbrainz-sytle metadata from a freedb entry would make this function perfect.

Reply:
Sure, will do.